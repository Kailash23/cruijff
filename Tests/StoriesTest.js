import initStoryshots from '@storybook/addon-storyshots';

console.error = (...args) => {
  throw new Error(args);
};

console.warn = (...args) => {
  throw new Error(args);
};

initStoryshots({
  // storyKindRegex: /^((?!.*?(PlanGameScreen|TimePickerField|SportDateTimeSlide|LocationSlide|OnboardingForm|OnboardingScreen)).)*$/,
});
