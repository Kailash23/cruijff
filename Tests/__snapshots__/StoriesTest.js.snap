// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Auth.LoginForm LoginForm 1`] = `null`;

exports[`Storyshots Auth.SignupForm SignupForm 1`] = `null`;

exports[`Storyshots Backgrounds.FieldBackground FieldBackground 1`] = `null`;

exports[`Storyshots Backgrounds.LogoHeaderBackground LogoHeaderBackground green theme 1`] = `null`;

exports[`Storyshots Backgrounds.LogoHeaderBackground LogoHeaderBackground white theme 1`] = `null`;

exports[`Storyshots Common.AlertMsg AlertMsg error 1`] = `null`;

exports[`Storyshots Common.AlertMsg AlertMsg info 1`] = `null`;

exports[`Storyshots Common.AlertMsg AlertMsg info long text 1`] = `null`;

exports[`Storyshots Common.AlertMsg AlertMsg success 1`] = `null`;

exports[`Storyshots Common.AlertMsg AlertMsg warning 1`] = `null`;

exports[`Storyshots Common.Avatar Avatar no props 1`] = `null`;

exports[`Storyshots Common.Avatar Avatar with TEXT 1`] = `null`;

exports[`Storyshots Common.Avatar Avatar with USER 1`] = `null`;

exports[`Storyshots Common.AvatarPicker AvatarPicker with USER 1`] = `null`;

exports[`Storyshots Common.Block Block 1`] = `null`;

exports[`Storyshots Common.Block Block 100px fixed height 1`] = `null`;

exports[`Storyshots Common.Block Block 100px fixed height bgColor gray 1`] = `null`;

exports[`Storyshots Common.Calendar Calendar 1`] = `null`;

exports[`Storyshots Common.CapacityPicker CapacityPicker 1`] = `null`;

exports[`Storyshots Common.CapacityPickerField CapacityPickerField 1`] = `null`;

exports[`Storyshots Common.CapacityPickerField CapacityPickerField boxed size ML 1`] = `null`;

exports[`Storyshots Common.CapacityPickerField CapacityPickerField white theme 1`] = `null`;

exports[`Storyshots Common.CapacityPickerField CapacityPickerField with ERROR 1`] = `null`;

exports[`Storyshots Common.Checkbox Checkbox 1`] = `null`;

exports[`Storyshots Common.Checkbox Checkbox white theme 1`] = `null`;

exports[`Storyshots Common.DatePickerField DatePickerField 1`] = `null`;

exports[`Storyshots Common.DatePickerField DatePickerField boxed size ML dateFormat DD/MM/YYYY 1`] = `null`;

exports[`Storyshots Common.DatePickerField DatePickerField white theme 1`] = `null`;

exports[`Storyshots Common.Divider Divider 1`] = `null`;

exports[`Storyshots Common.Divider Divider row 1`] = `null`;

exports[`Storyshots Common.DotSpacer DotSpacer default size M 1`] = `null`;

exports[`Storyshots Common.DotSpacer DotSpacer size L 1`] = `null`;

exports[`Storyshots Common.DotSpacer DotSpacer size S 1`] = `null`;

exports[`Storyshots Common.Dropdown Dropdown 1`] = `null`;

exports[`Storyshots Common.Dropdown Dropdown with error 1`] = `null`;

exports[`Storyshots Common.Dropdown Dropdown with white theme 1`] = `null`;

exports[`Storyshots Common.Dropdown Dropdown with white theme size ML 1`] = `null`;

exports[`Storyshots Common.DurationPickerField DurationPickerField 1`] = `null`;

exports[`Storyshots Common.DurationPickerField DurationPickerField boxed size ML 1`] = `null`;

exports[`Storyshots Common.DurationPickerField DurationPickerField white theme 1`] = `null`;

exports[`Storyshots Common.HeaderBtn HeaderBtn 1`] = `null`;

exports[`Storyshots Common.InputField InputField Dropdown comp 1`] = `null`;

exports[`Storyshots Common.InputField InputField Dropdown comp DISABLED 1`] = `null`;

exports[`Storyshots Common.InputField InputField Dropdown comp boxed 1`] = `null`;

exports[`Storyshots Common.InputField InputField Dropdown comp boxed DISABLED 1`] = `null`;

exports[`Storyshots Common.InputField InputField Dropdown comp boxed with ERROR 1`] = `null`;

exports[`Storyshots Common.InputField InputField Dropdown comp with ERROR 1`] = `null`;

exports[`Storyshots Common.InputField InputField TextField comp 1`] = `null`;

exports[`Storyshots Common.InputField InputField TextField comp DISABLED 1`] = `null`;

exports[`Storyshots Common.InputField InputField TextField comp boxed 1`] = `null`;

exports[`Storyshots Common.InputField InputField TextField comp boxed DISABLED 1`] = `null`;

exports[`Storyshots Common.InputField InputField TextField comp boxed with ERROR 1`] = `null`;

exports[`Storyshots Common.InputField InputField TextField comp with ERROR 1`] = `null`;

exports[`Storyshots Common.InputField InputField size ML 1`] = `null`;

exports[`Storyshots Common.InputField InputField size S 1`] = `null`;

exports[`Storyshots Common.InputField InputField white theme 1`] = `null`;

exports[`Storyshots Common.Link Link 1`] = `null`;

exports[`Storyshots Common.Link Link with icon 1`] = `null`;

exports[`Storyshots Common.Menu Menu 1`] = `null`;

exports[`Storyshots Common.NavBar NavBar 1`] = `null`;

exports[`Storyshots Common.NavBarButton NavBarButton 1`] = `null`;

exports[`Storyshots Common.NavDot NavDot active 1`] = `null`;

exports[`Storyshots Common.NavDot NavDot default 1`] = `null`;

exports[`Storyshots Common.NavDots Default 1`] = `null`;

exports[`Storyshots Common.RaisedButton RaisedButton default 1`] = `null`;

exports[`Storyshots Common.RaisedButton RaisedButton disabled 1`] = `null`;

exports[`Storyshots Common.RaisedButton RaisedButton ghost 1`] = `null`;

exports[`Storyshots Common.RaisedButton RaisedButton ghost disabled 1`] = `null`;

exports[`Storyshots Common.RaisedButton RaisedButton info 1`] = `null`;

exports[`Storyshots Common.RaisedButton RaisedButton primary 1`] = `null`;

exports[`Storyshots Common.RaisedButton RaisedButton primary small 1`] = `null`;

exports[`Storyshots Common.RaisedButton RaisedButton secondary 1`] = `null`;

exports[`Storyshots Common.RaisedButton RaisedButton warning 1`] = `null`;

exports[`Storyshots Common.RoundButton RoundButton dark 1`] = `null`;

exports[`Storyshots Common.RoundButton RoundButton default 1`] = `null`;

exports[`Storyshots Common.RoundButton RoundButton disabled 1`] = `null`;

exports[`Storyshots Common.RoundButton RoundButton ghost 1`] = `null`;

exports[`Storyshots Common.RoundButton RoundButton ghost disabled 1`] = `null`;

exports[`Storyshots Common.RoundButton RoundButton info 1`] = `null`;

exports[`Storyshots Common.RoundButton RoundButton primary 1`] = `null`;

exports[`Storyshots Common.RoundButton RoundButton primary reverse 1`] = `null`;

exports[`Storyshots Common.RoundButton RoundButton primary small 1`] = `null`;

exports[`Storyshots Common.RoundButton RoundButton secondary 1`] = `null`;

exports[`Storyshots Common.RoundButton RoundButton translucid 1`] = `null`;

exports[`Storyshots Common.RoundButton RoundButton warning 1`] = `null`;

exports[`Storyshots Common.RoundButton RoundButton warning reverse 1`] = `null`;

exports[`Storyshots Common.Row Row 1`] = `null`;

exports[`Storyshots Common.Slider Slider default 1`] = `null`;

exports[`Storyshots Common.Slider Slider initialValue 5 1`] = `null`;

exports[`Storyshots Common.SliderWithText SliderWithText 1`] = `null`;

exports[`Storyshots Common.Spacer Spacer size L 1`] = `null`;

exports[`Storyshots Common.Spacer Spacer size L row 1`] = `null`;

exports[`Storyshots Common.Spacer Spacer size M 1`] = `null`;

exports[`Storyshots Common.Spacer Spacer size M row 1`] = `null`;

exports[`Storyshots Common.Spacer Spacer size S 1`] = `null`;

exports[`Storyshots Common.Spacer Spacer size S row 1`] = `null`;

exports[`Storyshots Common.SportCard SportCard 1`] = `null`;

exports[`Storyshots Common.SportCard SportCard isSelected 1`] = `null`;

exports[`Storyshots Common.SportPickerField SportPickerField 1`] = `null`;

exports[`Storyshots Common.SportPickerField SportPickerField boxed size ML 1`] = `null`;

exports[`Storyshots Common.SportPickerField SportPickerField white theme 1`] = `null`;

exports[`Storyshots Common.SportsList SportsList 1`] = `null`;

exports[`Storyshots Common.SwitchWithText SwitchWithText 1`] = `null`;

exports[`Storyshots Common.Tag Tag error 1`] = `null`;

exports[`Storyshots Common.Tag Tag info 1`] = `null`;

exports[`Storyshots Common.Tag Tag info long text 1`] = `null`;

exports[`Storyshots Common.Tag Tag success 1`] = `null`;

exports[`Storyshots Common.Tag Tag warning 1`] = `null`;

exports[`Storyshots Common.TextField TextField 1`] = `null`;

exports[`Storyshots Common.TextField TextField disabled 1`] = `null`;

exports[`Storyshots Common.TextField TextField with characterRestriction = 20 1`] = `null`;

exports[`Storyshots Common.TextField TextField with error 1`] = `null`;

exports[`Storyshots Common.TextField TextField with white theme 1`] = `null`;

exports[`Storyshots Common.TextField TextField with white theme size ML 1`] = `null`;

exports[`Storyshots Common.TextField TextField with white theme with ERROR 1`] = `null`;

exports[`Storyshots DarkFooter DarkFooter NO Back button 1`] = `null`;

exports[`Storyshots DarkFooter DarkFooter default 1`] = `null`;

exports[`Storyshots DarkFooterButton DarkFooterButton BACK 1`] = `null`;

exports[`Storyshots DarkFooterButton DarkFooterButton NEXT 1`] = `null`;

exports[`Storyshots DarkFooterButton DarkFooterButton NEXT disabled 1`] = `null`;

exports[`Storyshots Games.AdminMenu AdminMenu 1`] = `null`;

exports[`Storyshots Games.Attendees Attendees 1`] = `null`;

exports[`Storyshots Games.CancelGameForm CancelGameForm 1`] = `null`;

exports[`Storyshots Games.CancelMsg CancelMsg 1`] = `null`;

exports[`Storyshots Games.ClickableAttendees ClickableAttendees 1`] = `null`;

exports[`Storyshots Games.EditGameForm EditGameForm 1`] = `null`;

exports[`Storyshots Games.GameCanceledFlag GameCanceledFlag 1`] = `null`;

exports[`Storyshots Games.GameDetails GameDetails CANCELED 1`] = `null`;

exports[`Storyshots Games.GameDetails GameDetails PLANNED 1`] = `null`;

exports[`Storyshots Games.GameListCard GameListCard CANCELED 1`] = `null`;

exports[`Storyshots Games.GameListCard GameListCard CANCELED short title/name 1`] = `null`;

exports[`Storyshots Games.GameListCard GameListCard PLANNED 1`] = `null`;

exports[`Storyshots Games.GameListCard GameListCard PLANNED short title/name 1`] = `null`;

exports[`Storyshots Games.GameProperties GameProperties 1`] = `null`;

exports[`Storyshots Games.GamesList GamesList 1`] = `null`;

exports[`Storyshots Games.ImageModal ImageModal CancelConfirmModal comp 1`] = `null`;

exports[`Storyshots Games.ImageModal ImageModal ConfirmModal comp 1`] = `null`;

exports[`Storyshots Games.OpenSpots OpenSpots 1`] = `null`;

exports[`Storyshots Games.Organizer Organizer M textSize 1`] = `null`;

exports[`Storyshots Games.Organizer Organizer S textSize 1`] = `null`;

exports[`Storyshots Games.Organizer Organizer default textSize (SM) 1`] = `null`;

exports[`Storyshots Games.ShareGameButton ShareGameButton default 1`] = `null`;

exports[`Storyshots ImageSwiper Default 1`] = `null`;

exports[`Storyshots Layouts.ClosableLayout ClosableLayout black theme 1`] = `null`;

exports[`Storyshots Layouts.ClosableLayout ClosableLayout white theme 1`] = `null`;

exports[`Storyshots Layouts.ClosableLayout ClosableLayout white theme NOT closable 1`] = `null`;

exports[`Storyshots Layouts.FixedTopLayout FixedTopLayout 1`] = `null`;

exports[`Storyshots Layouts.TopBottomLayout TopBottomLayout black theme 1`] = `null`;

exports[`Storyshots Logo Default 1`] = `null`;

exports[`Storyshots Modals.CancelConfirmModal CancelConfirmModal 1`] = `null`;

exports[`Storyshots Modals.CapacityPickerModal CapacityPickerModal 1`] = `null`;

exports[`Storyshots Modals.ConfirmModal ConfirmModal 1`] = `null`;

exports[`Storyshots Modals.DatePickerModal DatePickerModal 1`] = `null`;

exports[`Storyshots Modals.DialogModal DialogModal 1`] = `null`;

exports[`Storyshots Modals.DialogModal DialogModal big children 1`] = `null`;

exports[`Storyshots Modals.DialogModal DialogModal big children no header 1`] = `null`;

exports[`Storyshots Modals.Modal Modal no footer 1`] = `null`;

exports[`Storyshots Modals.Modal Modal no footer big children 1`] = `null`;

exports[`Storyshots Modals.Modal Modal no footer overflow children 1`] = `null`;

exports[`Storyshots Modals.SportPickerModal SportPickerModal 1`] = `null`;

exports[`Storyshots Modals.SpotPickerModal SpotPickerModal 1`] = `null`;

exports[`Storyshots NothingFound Default 1`] = `null`;

exports[`Storyshots Onboarding Default 1`] = `null`;

exports[`Storyshots PlanGame.DescriptionSlide DescriptionSlide white theme 1`] = `null`;

exports[`Storyshots PlanGame.InviteOnly InviteOnly isPrivate 1`] = `null`;

exports[`Storyshots PlanGame.InviteOnly InviteOnly isPublic 1`] = `null`;

exports[`Storyshots PlanGame.ShareGameForm ShareGameForm white theme 1`] = `null`;

exports[`Storyshots PlanGame.ShareLink ShareLink 1`] = `null`;

exports[`Storyshots PlanGame.ShareLink ShareLink long link 1`] = `null`;

exports[`Storyshots PlanGame.SpotSlide SpotSlide white theme 1`] = `null`;

exports[`Storyshots Profile.EditProfileForm EditProfileForm 1`] = `null`;

exports[`Storyshots Profile.ProfileDetails ProfileDetails 1`] = `null`;

exports[`Storyshots Profile.ProfileEditScreen ProfileEditScreen 1`] = `null`;

exports[`Storyshots Profile.ProfileTabs ProfileTabs 1`] = `null`;

exports[`Storyshots Profile.UserMenu UserMenu 1`] = `null`;

exports[`Storyshots Profile.UserSpots UserSpots 1`] = `null`;

exports[`Storyshots Rating 1 1`] = `null`;

exports[`Storyshots Rating 3 1`] = `null`;

exports[`Storyshots Rating 4.5 1`] = `null`;

exports[`Storyshots RatingBig 1 1`] = `null`;

exports[`Storyshots RatingBig 3 1`] = `null`;

exports[`Storyshots RatingBig 4.5 1`] = `null`;

exports[`Storyshots Screen.Auth LoginScreen 1`] = `null`;

exports[`Storyshots Screen.Auth SignupScreen 1`] = `null`;

exports[`Storyshots Screens SplashScreen 1`] = `null`;

exports[`Storyshots Screens.Auth LoggedOutScreen 1`] = `null`;

exports[`Storyshots Screens.Auth LoggedOutScreen closable 1`] = `null`;

exports[`Storyshots Screens.Debug DebugScreen 1`] = `null`;

exports[`Storyshots Screens.Games CancelGameScreen 1`] = `null`;

exports[`Storyshots Screens.Games EditGameScreen 1`] = `null`;

exports[`Storyshots Screens.Games GameDetailsScreen 1`] = `null`;

exports[`Storyshots Screens.Games GamesListScreen 1`] = `null`;

exports[`Storyshots Screens.Games PlayersListScreen 1`] = `null`;

exports[`Storyshots Screens.Info InfoScreen 1`] = `null`;

exports[`Storyshots Screens.Plan.ShareGameScreen ShareGameScreen 1`] = `null`;

exports[`Storyshots Screens.Spots SpotsFilterScreen 1`] = `null`;

exports[`Storyshots Screens.Spots SpotsListScreen 1`] = `null`;

exports[`Storyshots Screenslider ScreenSlider 1`] = `null`;

exports[`Storyshots Spots.BackgroundImage BackgroundImage 1`] = `null`;

exports[`Storyshots Spots.SpotHeader SpotHeader default 1`] = `null`;

exports[`Storyshots Spots.SpotHeader SpotHeader withDistance withGames 1`] = `null`;

exports[`Storyshots Spots.SpotHeader SpotHeader withDistance withGames gray 1`] = `null`;

exports[`Storyshots Spots.SpotImage SpotImage 1`] = `null`;

exports[`Storyshots Spots.SpotLink SpotLink 1`] = `null`;

exports[`Storyshots Spots.SpotListCard SpotListCard 1`] = `null`;

exports[`Storyshots Spots.SpotListCardSmall SpotListCardSmall active 1`] = `null`;

exports[`Storyshots Spots.SpotListCardSmall SpotListCardSmall default 1`] = `null`;

exports[`Storyshots Spots.SpotMap SpotMap 1`] = `null`;

exports[`Storyshots Spots.SpotPickerField SpotPickerField default 1`] = `null`;

exports[`Storyshots Spots.SpotProperties SpotProperties 1`] = `null`;

exports[`Storyshots Spots.SpotsFilterFlap SpotsFilterFlap 1`] = `null`;

exports[`Storyshots Spots.SpotsList SpotsList big card 1`] = `null`;

exports[`Storyshots Spots.SpotsList SpotsList small card 1`] = `null`;

exports[`Storyshots Text All sizes 1`] = `null`;

exports[`Storyshots Text Default text 1`] = `null`;

exports[`Storyshots Text Large text 1`] = `null`;

exports[`Storyshots Text Medium text 1`] = `null`;

exports[`Storyshots Text Small text 1`] = `null`;
